.github/workflows/v1_testing.yml

```diff
@@ -1,5 +1,6 @@
 name: v1 Protos Testing
 on:
+  pull_request:
```

Comment:
So... are we considering making this compulsory right now? If that's the case, maybe we need to add it to the main workflow instead. If it is expected to continue failing (temporarily) let's keep it here